### PR TITLE
CMake fixes and some cleanups

### PR DESCRIPTION
**Pull request by sshedi**
- Change minimum version of cmake required.
- Remove updateinfo timer & service, it should be taken care by motd
- Install `history-db-util` under `/usr/libexec/tdnf`, `libexec` is the dedicated location for such use cases.

### CODE REVIEW

File: tdnf.spec.in [hunk 1]
```diff
@@ -150,80 +157,61 @@ cmake \
 
 %make_build
 %make_build python
-
-%if 0%{?with_check}
```
Comment (oliverkurth) [inline tdnf.spec.in:154]: Why is this changed to unconditional?

**Review by oliverkurth — COMMENTED** [review]
Looks good to me (thank you!), just one simple question.

File: tdnf.spec.in (hunk 1 above)
Comment (sshedi) [reply tdnf.spec.in:154]: Yeah, better to keep it in this case. Thanks.
